src/calculateRentalCost.js

```diff
@@ -4,7 +4,9 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const off = days >= 7 ? 50 : days >= 3 ? 20 : 0;
```

Comment:
The variable name 'off' is not very descriptive. Consider using a more descriptive name like 'discount' to clarify its purpose. This aligns with the checklist item: 'Make names descriptive, so they explain the purpose of variables and functions.'